faker/providers/internet/__init__.py

```diff
@@ -115,6 +115,13 @@ class Provider(BaseProvider):
         '?txtsize=55&txt={width}x{height}&w={width}&h={height}',
         'https://www.lorempixel.com/{width}/{height}',
         'https://dummyimage.com/{width}x{height}',
+        'https://baconmockup.com/{width}/{height}',
+        'https://loremflickr.com/{width}/{height}',
+        'https://placekitten.com/{width}/{height}',
+        'https://www.placecage.com/{width}/{height}',
+        'https://placeimg.com/{width}/{height}/any',
+        'https://placebear.com/{width}/{height}',
```

Comment:
Same concern as for `placecage.com`

Reply:
I've removed most of the new URLs, just to be safe.